target/dovecot/10-master.conf

```
@@ -100,7 +100,7 @@ service auth {
   }
 
   # Postfix smtp-auth
-  unix_listener /var/spool/postfix/private/auth {
+  unix_listener /var/tmp/sasl-auth.socket {
     mode = 0666
     user = docker
     group = docker
```

I'd like to see this also being changes to `postfix:postfix`, i.e.

```suggestion
    user = postfix
    group = postfix
```

---

I think it should be `postfix` yes. The permissions are meant to be `660` for that AFAIK, but if we're allowing anyone to read it, that might explain why it hasn't mattered with `docker` in the past, and possibly why some users have had permission errors in the past if it was corrected :sweat_smile:

---

I, I overlooked that; indeed! Permissions should be `0660` of course!!

---

Solved in [2d06c07](https://github.com/docker-mailserver/docker-mailserver/pull/3125/commits/2d06c07689b597e5b8148cd9357a5ac4a3ccbafd)